apollo-portal/src/main/resources/static/index.html

```diff
@@ -36,37 +36,221 @@
+                <h5>{{'Index.RecentlyViewedItems' | translate }}</h5>
+            </div>
+        </div>
+        <div class="main-table">
+            <div class="create-app-list" ng-show="whichContent==='1'">
+                <div style="width: 180px;margin-left: 30px" ng-click="goToCreateAppPage()"
+                     ng-if="hasCreateApplicationPermission">
+                    <div href="#" class="thumbnail create-btn hover cursor-pointer"
+                         style="display: flex;justify-content: center;align-items: center">
+                        <div><img src="img/plus-white.png" /></div>
+                        <div style="margin-left: 5px"><h5>{{'Index.CreateProject' | translate }}</h5></div>
+                    </div>
+                </div>
+                <div class="display-table" ng-show="createdApps.length > 0">
+                    <table>
+                        <tr>
+                            <th style="width: 20%">{{'Index.appTable.appId' | translate }}</th>
+                            <th style="width: 30%">{{'Index.appTable.appName' | translate }}</th>
+                            <th style="width: 20%">{{'Index.appTable.CreatedTime' | translate }}</th>
+                            <th style="width: 30%">{{'Index.appTable.orgName' | translate }}</th>
+                        </tr>
+                        <tr ng-repeat="app in createdApps" ng-click="goToAppHomePage(app.appId)" href="#" class="hover cursor-pointer">
+                            <td style="width: 20%">{{ app.appId }}</td>
+                            <td style="width: 30%">{{ app.name }}</td>
+                            <td style="width: 20%">{{ app.dataChangeCreatedTime.slice(0,10) }}</td>
+                            <td style="width: 30%">{{ app.orgName }}</td>
+                        </tr>
+                    </table>
+                </div>
+                <div ng-show="!hasMoreCreatedApps" style="height: 15px"></div>
+                <div style="width: 180px;margin-left: 30px;margin-top: 20px" ng-show="hasMoreCreatedApps"
+                     ng-click="getUserCreatedApps()">
+                    <div href="#" class="thumbnail hover cursor-pointer"
+                         style="display: flex;justify-content: center;align-items: center">
+                        <div><img class="more-img" src="img/more.png" /></div>
+                        <div style="margin-left: 5px"><h5>{{'Index.LoadMore' | translate }}</h5></div>
+                    </div>
+                </div>
+            </div>
+            <div class="favorites-app-list" ng-show="whichContent==='2'">
+                <div class="display-table" ng-show="favorites && favorites.length > 0">
+                    <table>
+                        <tr>
+                            <th style="width: 20%">{{'Index.appTable.appId' | translate }}</th>
```

Comment:
The same as above, it's better to display the information as those in the project page: App Id, App Name, Department, App Owner, Email.